### PR TITLE
Migrations 2

### PR DESCRIPTION
**Pull request by brennanjl**
Using this as the PR for updating the migrations tutorial.

### CODE REVIEW

File: docs/node/migrations/zero-downtime-migrations/index.mdx
Comment (brennanjl) [inline docs/node/migrations/zero-downtime-migrations/index.mdx:125]: I used an online diff checker to ensure that these are the same. IDK why git is saying these are different

**Comment by brennanjl** [comment]
Once someone thinks this is good to go, I will overwrite the old tutorial

File: docs/node/migrations/2-zero-downtime-migrations/index.mdx
Comment (charithabandi) [inline docs/node/migrations/2-zero-downtime-migrations/index.mdx:98]: broken link

**Review by charithabandi — APPROVED** [review]
lgtm!

**Comment by brennanjl** [comment]
Will wait for Luke's go-ahead before merging

**Pull request merged**
